comms/EclMeta.cpp

```diff
@@ -315,6 +318,13 @@ void CEclMeta::PopulateMeta(const boost::filesystem::wpath & fileOrDir, const st
     }
 }
 
+CEclFile *CEclMeta::CreateFile(const std::_tstring & dottedPath, const std::_tstring & path)
+{
+    CEclFile *file = new CEclFile(path);
+    m_masterMeta[dottedPath] = file;
+    return file;
+}
```

Comment:
Need to ensure that this is placed into a link counted container so it doesn't leak?

Reply:
(just checked and it is.)

Reply:
@GordonSmith yes. You are right.